components/automate-ui/src/app/services/chef-session/chef-session.service.ts

```diff
@@ -146,11 +146,14 @@ export class ChefSessionService implements CanActivate {
     return !isNull(localStorage.getItem(sessionKey));
   }
 
-  logout(url: string = '/', refresh: boolean = false): void {
+  // url: UI route to go back to when the (next) signin process has succeeded
+  // noHint: for the sign in, don't try to skip the method selection
+  logout(url?: string, noHint?: boolean): void {
```

Comment:
👉 The crucial bit is that the http auth interceptor **hasn't changed**: [we're still calling `logout()` there](https://github.com/chef/automate/blob/8a5d470bcb929cf6d6ca88fdc2f120cb1e35dd5b/components/automate-ui/src/app/services/http/http-client-auth.interceptor.ts#L43), but it means something slightly different now.